### PR TITLE
[ESIMD] Add test for radix sort.

### PR DESCRIPTION
**Pull request by kychendev**

### CODE REVIEW

File: SYCL/ESIMD/radix_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+    q.submit([&](handler &cgh) {
+      cgh.parallel_for<class Accum_final>(
+          range<2>{1, 1} * range<2>{1, 1}, [=](item<2> it) SYCL_ESIMD_KERNEL {
+            cmk_acum_final(buf, it.get_id(0), elem_stride, n_entries);
+          });
+    });
+    return;
+  }
+
+  // std::cout << "*** n_entries: " << n_entries << " elem_stide: " <<
```
Comment (kbobrovs) [inline SYCL/ESIMD/radix_sort.cpp]: Nit: maybe cleanup here and line 359

**Review by kbobrovs — DISMISSED** [review]

File: SYCL/ESIMD/radix_sort.cpp (hunk 1 above)
Comment (kychendev) [reply SYCL/ESIMD/radix_sort.cpp]: The commented lines were added for debugging purpose - removed.

**Review by kbobrovs — APPROVED** [review]

**Comment by kbobrovs** [comment]
@kychendev , please provide info on failing tests so that this PR could be merged if failures are unrelated.

**Comment by kychendev** [comment]
This change only contains a new radix_sort kernel, which has passed CI test.

The following new failed cases should be irrelevant to this PR:

[2022-09-20T01:59:16.620Z] New failures found: ['SYCL::Config/kernel_from_file.cpp', 'SYCL::DeviceLib/separate_compile_test.cpp', 'SYCL::SeparateCompile/test.cpp']

**Comment by kychendev** [comment]
@kbobrovs , could you help merge as the failing tests are unrelated.

**Pull request merged**